### ISSUE TITLE
(cont. from #166) Map Page: Update top row of buttons UI to overlap the map w/ the relevant functionalities

### ISSUE THREAD

**Issue by neozhixuan** [bettersg/recyclegowhere#169]
**Context**
Once #166 is done, we need to have the buttons at the top overlap the map, and follow the UI designs made by the UX team.
![image](https://github.com/bettersg/recyclegowhere/assets/79783660/e618eae3-f0c6-4cfc-9e41-2d4c76b700ab)

**Figma Link**
https://www.figma.com/file/kN115h4H1jk797qa3x08Et/RGW-Design_Master-Design-File?type=design&node-id=5304-15885&mode=design&t=1Qzyuj9RvjAVBST3-0

**Acceptance Criteria**

**Comment by nipuna-g** [comment]
@neozhixuan I have a few questions about this ticket. 

- Is the scope of this ticket to just change the UI components on the top of the page?

The new design doesn't seem to have equivalents to the following items:

![image](https://github.com/bettersg/recyclegowhere/assets/5859290/6723d591-5912-4b17-bc58-654f286d80c7)

1 - View Results
2,3 - We have two ways to filter results. 

Is this expected, and are we going to use the unified filter?
Can I assume that the changes for the popup filter and cards are out of scope?

Also, do we have a ticket to make the changes for the pill elements in the filter? Or is that part of the tickets?

Thanks 🙏🏾

**Comment by neozhixuan** [comment]
@nipuna-g you are right, apologies for not making it clear!

The "View Results" button is gone; "Edit" becomes the "<-" button, and the Filter button will have its UI updated to the "button with the down arrow and hamburger" (can see it on the right of the multiselect box). 

Let me know if you have any queries!

**Comment by nipuna-g** [comment]
Sounds good! It's all clear now. I'll finish the implementation for this today and create a PR.
